src/PowerShellEditorServices/Session/RunspaceDetails.cs

```diff
@@ -218,7 +218,16 @@ internal bool HasCapability<TCapability>()
                     // ProcessId property isn't on the object, move on.
                 }
 
-                if (runspace.ConnectionInfo.ComputerName != "localhost")
+                // Grab the $host.name which will tell us if we're in a PSRP session or not
+                string hostname =
+                        PowerShellContext.ExecuteScriptAndGetItem<string>(
+                            "$host.name",
+                            runspace,
+                            defaultValue: "");
+
+                // hostname is 'ServerRemoteHost' when the user enters a session.
+                // ex. Enter-PSSession, Enter-PSHostProcess
+                if (hostname == "ServerRemoteHost")
```

Comment:
```suggestion
                if (hostName.Equals("ServerRemoteHost", StringComparison.Ordinal))
```

I know it's functionally the same, but more explicit.